nightly-examples/examples/echo.rs

```diff
@@ -0,0 +1,89 @@
+//! A "hello world" echo server with Tokio
+//!
+//! This server will create a TCP listener, accept connections in a loop, and
+//! write back everything that's read off of each TCP connection.
+//!
+//! Because the Tokio runtime uses a thread pool, each TCP connection is
+//! processed concurrently with all other TCP connections across multiple
+//! threads.
+//!
+//! To see this server in action, you can run this in one terminal:
+//!
+//!     cargo run --example echo
+//!
+//! and in another terminal you can run:
+//!
+//!     cargo run --example connect 127.0.0.1:8080
```

Comment:
We don't have the `connect` example from `tokio`. What do you think about changing this use the `proxy_server` example as a client instead?
```suggestion
//!     cargo +nightly run --example proxy_server 127.0.0.1:8081 127.0.0.1:8080 
```

Reply:
I missed chaning the doc strings in general. That was sloppy, apologies.